src/tags/object/Paragraphs/Phrases.js

```diff
@@ -68,7 +175,12 @@ export const Phrases = observer(({ item, playingId, activeRef }) => {
           <span className={cls?.name} data-skip-node="true" style={style?.name}>{v[item.namekey]}</span>
         )}
 
-        <span className={cls?.text}>{v[item.textkey]}</span>
+        <span className={`${cls?.text} ${isFF(FF_LSDV_E_278) && styles.text}`}>
+          {v[item.textkey]}
+          {(isFF(FF_LSDV_E_278) && isActive) && (
+            <span ref={readingLineRef} className={`${styles.readingLine} reading-line`}></span>
+          )}
```

Comment:
```tsx
          {(isFF(FF_LSDV_E_278) && isActive) && (
            <span ref={readingLineRef} className={`${styles.readingLine} reading-line`}></span>
          )}
```
This is being added within the text content which the Paragraph parser is going to be influenced by, and likely is what was causing errors.

We have to make sure it is not counted within the paragraph data by either moving out of the span that also holds the text segment, or adding `data-skip-node="true"` to the reading line span.